layouts/community/60_ansi/stanrc85-ansi/config.h

```diff
@@ -0,0 +1,4 @@
+#define TAPPING_TERM 200
```

Comment:
This should/needs to have an include guard.

```suggestion
#pragma once

#define TAPPING_TERM 200
```

Reply:
Changes made.